### ISSUE TITLE
term_args not working on taxonomy fields

### ISSUE THREAD

**Issue by wearehyphen** [bootsz/wp-advanced-search#206]
Firstly, thanks for the great work so far, we've used this quite extensively over the last year and we love it.

We're currently building a site for a client, and we're having trouble getting `term_args` to work on a taxonomy field.

Basically we're setting up a basic filter based on categories, however we are using a 'hidden' category (`Featured - ID: 6`) to establish a different set of styles on featured posts, and we also don't want the default `Uncategorised - ID: 1` to show in the the list of checkboxes on the filter.

I found `term_args`, and thought that'll do the trick — however even the example code doesn't seem to affect the result at all. For now I can just hide the terms in question using CSS, however that's not ideal — and if there's an issue for more people, I thought I would flag it.

I've also tested it on a different site using the same setup (only on a custom taxonomy) and it has the same problem — `term_args` does nothing.

The code to set up the form here:

```php
function plus_category_filter() {
		$args = array();
		$args['wp_query'] = array(
			 'post_type' => 'post'
			,'posts_per_page' => 3
		);

		$args['form'] = array(
			'auto_submit' => true
		);

		$args['form']['ajax'] = array(
			 'enabled'	=> true
			,'show_default_results' => true
			,'results_template'  => 'wpas-ajax-results.php'
			,'loading_img'	=> get_template_directory_uri() . '/images/loader-light.png'
			,'url_hash' => false
		);

		$args['fields'][] = array(
			 'type' => 'taxonomy'
			,'taxonomy' => 'category'
			,'format'	=> 'checkbox'
			,'operator' => 'IN'
			,'term_args' => array( 'exclude' => 1,6 )
		);

		// $args['debug'] = true;
		// $args['debug_level'] = 'verbose';

		register_wpas_form( 'category-filter',$args );
	}
	add_action( 'init','plus_category_filter' );
```

Let me know if you need anything else to have a look into this!

**Comment by bootsz** [comment]
This issue should be fixed now in the latest patch (1.4.3).  Please re-download the code and give it a try.

**Comment by wearehyphen** [comment]
Thanks! Working perfectly now.

Just a note that in order for exclude to work, I had to put the ID's in an array;

```php
'term_args' => array( 'exclude' => array( 1,6 ) )
```
